data-structures/deque.md

```diff
@@ -14,6 +15,7 @@
 - [JavaScript 7 - Object-Oriented Programming](/javascript/javascript-7-oop.md)
 - [Runtime Complexity](/runtime-complexity/runtime-complexity.md)
 
+
 ### Motivation
 Deque is used as an abstract data type, also referenced as a double-ended queue.
 Similar to a queue, deques exist as an ordered collection of items. A deque's nature is unrestrictive, meaning that it easily allows for adding and removing items at either end. Deques are a unique hybrid linear structure that provide the capabilities of stacks and queues in a unique data structure. Deques can be implemented with a dynamic array or with a doubly-linked list and they do not require Last In First Out (LIFO) or FIFO (First In First Out) orderings that are required by both stacks and queues. The homophone "dequeue" is used as a verb, meaning to remove from a queue.
```

Comment:
FIFO (First In First Out) -> First In First Out (FIFO)

Reply:
Please do the review. I have made the change.